content_tmp/ansible/remove-subnet.yml

```diff
@@ -0,0 +1,14 @@
+---
+# Playbook Inputs:
+# - id (string) entity id
+# - kind (string) 'L2' for L2 subnets and 'L3' for L3 subnets (default)
+
+- name: Remove Subnet
+  hosts: localhost
+  connection: local
+  gather_facts: False
+  vars:
+    type: L3
```

Comment:
should this be called ```kind```? I don't see ```type``` anywhere else

Reply:
Thanks! Was struggling between `kind` and `type` so much that obviously I mixed it up eventually 😎